### DISCUSSION TITLE
Try syncthing core 1.27.8 on TestFlight

### DISCUSSION THREAD

**Discussion by SimonPickup** [MobiusSync/MobiusSync#156] [Beta]
Beta versions based on Syncthing 1.27.7 core will shortly be available on TestFlight. You can sign up now at:

- Möbius Sync: https://testflight.apple.com/join/n5A7ceuK
- Möbius Sync Pro: https://testflight.apple.com/join/TzJw0gsD

**Please note this is beta software and is not production ready.**

You should only use the software in an environment where you can tolerate data corruption or data loss.

Follow this thread, and see https://github.com/MobiusSync/MobiusSync/milestone/8 for more details on what is included etc.

**Comment by yingzhu146** [comment]
Very nice! just tried to test but

1. mobius sync - seems to work, but stops after 20mb
2. mobius sync pro - says beta is not accepting testers. 

I have mobius sync pro installed on iphone/ipad, but "restore purchase" throws an error ("Unable to restore purchase"). When I try to purchase it shows "Purchase on another device" (purchasing is currently not availabl on this device in your country or region).

My apple ID/location clearly says it's United States, just checked on all devices (iPhone, Mac, Vision Pro).

Interestingly enough, the currency is also incorrect (it shows in Euro, I used to have this account set to Germany at some point, but it's 100% set to US for years and I just checked again on all my devices and it says apple ID is US).

**Comment by SimonPickup** [comment]
Thanks for your feedback.

Mobius Sync Pro is still pending Apple review before it is released to TestFlight. Will update here once ready.

You should be able to purchase on Mobius Sync, as it runs in a testing sandbox and will not charge you anything. (I can't explain the currency mixup though.)

**Reply by SimonPickup** [reply]
Pro is now available via TestFlight

**Reply by yingzhu146** [reply]
amazing, so far so good, seems to work now! :)

**Reply by SimonPickup** [reply]
@yingzhu146 have you tried it on Vision Pro?

**Comment by SimonPickup** [comment]
New TestFlight build 202405310620 is out

Earlier builds:
- #146
- #141

This build:
- #125
- #111
- #110

**Comment by SimonPickup** [comment]
New TestFlight build 202406150620 is out

Earlier builds:
- #146
- #141
- #125
- #111
- #110

This build:
- #148

**Comment by SimonPickup** [comment]
We have now updated to Syncthing core 1.27.8 and released new TestFlight build 202406152020.

This build:
- #166
- #165

Earlier builds:
- #146
- #141
- #125
- #111
- #110
- #148

**Comment by SimonPickup** [comment]
New TestFlight build 202406220620 is out:

This build:
- #168

Earlier builds:
- #166
- #141
- #125
- #111
- #110
- #148
- #165

**Comment by SimonPickup** [comment]
This is now released publicly to the App Store. Thanks to all TestFlight testers!
